Samples/GroupGridIndices/Program.cs

```diff
@@ -70,33 +70,28 @@ static void WriteToGlobalIndex(ArrayView<int> dataView, int constant)
         static void Main()
         {
             // Create main context
-            using (var context = Context.CreateDefault())
+            using var context = Context.CreateDefault();
+
+            // For each available device...
+            foreach (var device in context)
             {
-                // For each available device...
-                foreach (var device in context)
-                {
-                    // Create accelerator for the given device
-                    using (var accelerator = device.CreateAccelerator(context))
-                    {
-                        Console.WriteLine($"Performing operations on {accelerator}");
+                // Create accelerator for the given device
+                using var accelerator = device.CreateAccelerator(context);
+                Console.WriteLine($"Performing operations on {accelerator}");
 
-                        var groupSize = accelerator.MaxNumThreadsPerGroup;
-                        KernelConfig kernelConfig = (2, groupSize);
+                var groupSize = accelerator.MaxNumThreadsPerGroup;
+                KernelConfig kernelConfig = (2, groupSize);
 
-                        using (var buffer = accelerator.Allocate1D<int>(kernelConfig.Size))
-                        {
-                            var groupedKernel = accelerator.LoadStreamKernel<ArrayView<int>, int>(GroupedKernel);
-                            groupedKernel(kernelConfig, buffer.View, 64);
+                using var buffer = accelerator.Allocate1D<int>(kernelConfig.Size);
+                var groupedKernel = accelerator.LoadStreamKernel<ArrayView<int>, int>(GroupedKernel);
+                groupedKernel(kernelConfig, buffer.View, 64);
 
-                            accelerator.Synchronize();
+                accelerator.Synchronize();
```

Comment:
nit: should we remove the explicit synchronization here and place a note that the synchronization happens implicitly by a call to `GetAsArray1D()`?